index.md

```diff
@@ -0,0 +1,224 @@
+---
```

Comment:
I think this file is not intended to be here?

Reply:
Same for `hellowrold.png`?

Reply:
And other files in root?

Reply:
removed